TwilioVerifySDKTests/TwilioSecurity/Sources/Keychain/KeychainQueryTests.swift

```diff
@@ -137,19 +133,17 @@ class KeychainQueryTests: XCTestCase {
     let query = keychainQuery.getData(withKey: Constants.alias)
     let keyClass = query[kSecClass] as! CFString
     let label = query[kSecAttrAccount] as! String
-    let access = query[kSecAttrAccessible] as! CFString
     let accessGroup = query[kSecAttrAccessGroup] as? String
 
     XCTAssertEqual(keyClass, kSecClassGenericPassword)
     XCTAssertEqual(label, Constants.alias)
-    XCTAssertEqual(access, kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly)
     XCTAssertNil(accessGroup)
   }
 
   func testSaveItem_withAccessGroup_shouldReturnValidQuery() {
     keychainQuery = KeychainQuery(accessGroup: Constants.accessGroup)
     let expectedData = "data".data(using: .utf8)!
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/1230259fcee173b2bad2b801de0a522beecabdd9/TwilioVerifySDKTests/TwilioSecurity/Sources/Keychain/KeychainQueryTests.swift#L145"></span>Prefer non-optional <code>Data(_:)</code> initializer when converting <code>String</code> to <code>Data</code><br />
<code>non_optional_string_data_conversion</code> <code>KeychainQueryTests.swift:145</code></td>
    </tr>
  </tbody>
</table>